<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectPlayers</class>
 <widget class="QDialog" name="selectPlayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The A&#038;T Four</string>
  </property>
  <widget class="QComboBox" name="boxP1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Ezell Blair, Jr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Franklin McCain</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Joseph McNeil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>David Richmond</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelP1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelP2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 211, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelP3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 212, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelP4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 154, 6)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt; Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Player Selection</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="playersText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Number of Players:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_totalP">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>6</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Timer Select:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tBox_P1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tBox_P2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tBox_P3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tBox_P4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="timerP1Text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>P1</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerP2Text">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>P2</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerP3Text">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>P3</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="timerP4Text">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>P4</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cSelectText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Character Select:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxP2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Ezell Blair, Jr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Franklin McCain</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Joseph McNeil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>David Richmond</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="boxP3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Ezell Blair, Jr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Franklin McCain</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Joseph McNeil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>David Richmond</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="boxP4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Ezell Blair, Jr.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Franklin McCain</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Joseph McNeil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>David Richmond</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="nextB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next &gt;&gt;</string>
   </property>
  </widget>
  <zorder>bgLabel</zorder>
  <zorder>boxP1</zorder>
  <zorder>labelP1</zorder>
  <zorder>labelP2</zorder>
  <zorder>labelP3</zorder>
  <zorder>labelP4</zorder>
  <zorder>backB</zorder>
  <zorder>title</zorder>
  <zorder>playersText</zorder>
  <zorder>box_totalP</zorder>
  <zorder>timerText</zorder>
  <zorder>timerP1Text</zorder>
  <zorder>timerP2Text</zorder>
  <zorder>timerP3Text</zorder>
  <zorder>timerP4Text</zorder>
  <zorder>tBox_P1</zorder>
  <zorder>tBox_P2</zorder>
  <zorder>tBox_P3</zorder>
  <zorder>tBox_P4</zorder>
  <zorder>cSelectText</zorder>
  <zorder>boxP2</zorder>
  <zorder>boxP3</zorder>
  <zorder>boxP4</zorder>
  <zorder>nextB</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
